pyanaconda/payload/dnf/payload.py

```diff
@@ -1257,6 +1263,54 @@ def pre_install(self):
         elif groupid:
             log.warning("Platform group %s not available.", groupid)
 
+    def _set_up_fips(self):
+        """Set up FIPS in the target system.
+
+        Copy the crypto policy from the installation environment
+        to target system the before the package installation. The
```

Comment:
The "the" wandered a bit from the place it should be ;-) Same for commit message.

Reply:
Fixed.